### PR TITLE
Feature/updated faq

### PR DESCRIPTION
**Pull request by peterchou3788**
https://github.com/TheOdinProject/theodinproject/issues/1192

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -58,15 +58,17 @@ def faq_items
...
         answer:
-          "<p>The <a href='courses/web-development-101'>Web Development 101 course</a> will give you a chance to explore several of the languages and technologies you’ll need to know on your journey to becoming a web developer.  Web Development 101 introduces HTML, CSS, Javascript, Ruby, Rails, Git, databases and more. We’ve also created mini-projects that give you a chance to test your new knowledge by building your own applications and websites!</p>
+          "<p> The Odin Project contains three curriculums for students to choose from: <a href='tracks/full-stack-ruby-on-rails'>Fullstack Ruby on Rails</a>, <a href='tracks/full-stack-javascript'>Fullstack Javascript</a>, and <a href='tracks/front-end-only'>Frontend only</a>. The Ruby on Rails track gives a general overview of what fullstack is. This is the default track for most beginners. The Javascript track focuses more on the connection between the webpage and the data with Javascript and NodeJS. The Frontend track primarily goes over HTML, CSS and Javascript but no Ruby on Rails. Below are some courses that are included in these curriculums:</p>
```
Comment (KevinMulhern) [inline app/helpers/application_helper.rb:63]: "The Odin Project contains three curriculums for students to choose from:"

Can we change "curriculums" to "tracks" in this  sentence please @peterchou3788? just to keep the terminology consistent throughout the site.

Comment (peterchou3788) [reply app/helpers/application_helper.rb:63]: Sure!

Comment (peterchou3788) [reply app/helpers/application_helper.rb]: I have updated it to your suggestion. Is there anything else you would like me to do?

**Comment by KevinMulhern** [comment]
Looks great @peterchou3788, thanks for making that change :)

**Pull request merged**